GVRf/Sample/gvr-sample/src/org/gearvrf/sample/SampleCubeScript.java

```diff
@@ -172,4 +180,52 @@ public void onStep() {
             }
         }
     }
+
+    // mode 0: center eye; mode 1: left eye; mode 2: right eye
+    public void captureScreen(int mode, String filename) {
+        switch (mode) {
+        case 0:
+            mGVRContext.captureScreenCenter(newScreenshotCallback(filename));
+            break;
+        case 1:
+            mGVRContext.captureScreenLeft(newScreenshotCallback(filename));
+            break;
+        case 2:
+            mGVRContext.captureScreenRight(newScreenshotCallback(filename));
+            break;
+        }
+    }
+
+    private GVRScreenshotCallback newScreenshotCallback(final String filename) {
+        return new GVRScreenshotCallback() {
+
+            @Override
+            public void onScreenCaptured(Bitmap bitmap) {
+                if (bitmap != null) {
+                    File file = new File(
+                            Environment.getExternalStorageDirectory(), filename);
+                    FileOutputStream outputStream = null;
+                    try {
+                        outputStream = new FileOutputStream(file);
+                    } catch (FileNotFoundException e) {
+                        e.printStackTrace();
```

Comment:
If you get an exception, the "file cannot be opened for writing" - so you shouldn't continue!